src/js/modules/GlobalSearch.js

```diff
@@ -0,0 +1,23 @@
+import React, {Component} from 'react';
+
+class GlobalSearch extends Component {
+
+  render() {
+
+    return  (
+
+      <div>
```

Comment:
super minor for accessibility, but USWDS has a role on the wrapper div.
`<div role="search"><form class="usa-search usa-search-big">...</form></div>`

Reply:
Let me know if you need any help with this @cthibaultatx